streams/id.m3u

```diff
@@ -13,60 +57,235 @@ http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
+https://wowza.convergen.co/gprtv/myLive/playlist.m3u8
 #EXTINF:-1 tvg-id="GTV.id" status="online",GTV (720p)
 #EXTVLCOPT:http-referrer=https://www.rctiplus.com/
 https://d322b885qvsbxg.cloudfront.net/GTV2021.m3u8
+#EXTINF:-1 tvg-id="HopeChannelIndonesia.id" status="error",Hope Channel Indonesia (1080p) [Not 24/7]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://m.youtube.com/c/HopeChannelIndonesia/live
+#EXTINF:-1 tvg-id="HumaBetangTV.id" status="error",Huma Betang TV (720p) [Not 24/7]
+https://v3.siar.us/humabetangtv/live/playlist.m3u8
 #EXTINF:-1 tvg-id="IAmChannel.id" status="online",I AM CHANNEL (576p) [Not 24/7]
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="Indosiar.id" status="error",Indosiar (720p) [Geo-blocked]
 http://rr2.dens.tv/s/s04/01.m3u8
 #EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
 #EXTVLCOPT:http-referrer=https://www.rctiplus.com/
 https://d2hfpzcndkyscp.cloudfront.net/INEWS_2021.m3u8
+#EXTINF:-1 tvg-id="InspiraTV.id" status="error",Inspira TV (720p) [Not 24/7]
+https://inspiratv.siar.us/inspiratv/live/playlist.m3u8
+#EXTINF:-1 tvg-id="IzzahTV.id" status="error",Izzah TV (480p)
+https://streaming.radiosalamjambi.com/izzahtv.m3u8
+#EXTINF:-1 tvg-id="JTV.id" status="error",JTV (1080p) [Not 24/7]
+https://cdn02.mola.tv/Content/HLS/Live/channel(f2cd666d-f350-1140-7ce5-3d7f39f84727)/08.m3u8
+#EXTINF:-1 tvg-id="JTV.id" status="error",JTV (720p) [Not 24/7]
+http://122.248.43.138:1935/ch2/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="JakTV.id" status="error",Jak TV (480p) [Geo-blocked]
+https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(7a73ed02-e0de-4df1-8da3-5abfd75f24bd)/index.m3u8
+#EXTINF:-1 tvg-id="JakTV.id" status="error",Jak TV (360p)
+https://bravo.kugo.id/jaktv/udp.stream_360p/playlist.m3u8
+#EXTINF:-1 tvg-id="JambiTV.id" status="error",Jambi TV (480p) [Not 24/7]
+http://wowza58.indostreamserver.com:1935/jambitv/live/playlist.m3u8
+#EXTINF:-1 tvg-id="JambiTV.id" status="error",Jambi TV (576p) [Not 24/7]
+http://122.248.43.138:1935/ch23/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="JawaPosTV.id" status="error",Jawa Pos TV (480p) [Not 24/7]
+https://v2.siar.us/jawapostv/live/chunks.m3u8
+#EXTINF:-1 tvg-id="JayaTV.id" status="error",Jaya TV (360p) [Geo-blocked]
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
+https://h1.intechmedia.net/intech/ch52.m3u8
+#EXTINF:-1 tvg-id="JekTV.id" status="error",Jek TV (360p) [Not 24/7] 
+http://128.199.90.131:1935/jektv/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="JekTV.id" status="error",Jek TV (360p) [Not 24/7] 
+http://122.248.43.138:1935/ch14/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="JogjaIstimewaTV.id" status="error",Jogja Istimewa TV (720p)
+http://103.255.15.222:1935/tv/jitv_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="JogjaIstimewaTV.id" status="error",Jogja Istimewa TV (360p) [Geo-blocked]
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
+https://h1.intechmedia.net/intech/ch65.m3u8
```

Comment:
CH78 has something... however can't seem to see anything.

Reply:
https://h1.intechmedia.net/intech/ch78.m3u8 = TVRI

Reply:
Oh wait, that's TVRI East Java. I was wrong. Sorry

Reply:
It's very easy to find the link for these streams anyways, visit https://youtv.live/, then view source of the page. Remember to use browser user agent and no referrer or referrer set to"https://youtv.live/" while playing it on VLC. Otherwise you will be redirected to YouTV app installation procedure (especially links that starts with "https://h1.intechmedia.net/nasional/").

Reply:
Assuming the is no problem in this review anymore